py3status/modules/coin_market.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+"""
+Display cryptocurrency data.
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module. A message from the site:
+        Please limit requests to no more than 10 per minute.
+        (default 600)
+    format: display format for this module (default '{format_coin}')
+    format_coin: display format for coins
+        (default '[\?color=degraded {name}] ${price_usd:.2f} {percent_change_24h}')
+    format_separator: show separator only if more than one (default ' ')
+    markets: comma separated list of supported markets https://coinmarketcap.com
+        (default ['btc'])
+    request_timeout: time to wait for a response, in seconds (default 5)
+
+Format placeholder:
+    {format_coin} format for cryptocurrency coins
+
+format_coin placeholders:
+    {24h_volume_usd}      eg 1435150000.0
+    {available_supply}    eg 16404825.0
+    {id}                  eg bitcoin
+    {last_updated}        eg 1498135152
+    {market_cap_usd}      eg 44119956596.0
+    {name}                eg Bitcoin
+    {percent_change_1h}   eg -0.17%
+    {percent_change_24h}  eg -1.93%
+    {percent_change_7d}   eg +14.73%
+    {price_btc}           eg 1.0
+    {price_usd}           eg 2689.45
```

Comment:
>can the do that? what if I want EUR, GBP etc I don't think you can at the moment. If it got them all for me then that'd be nice but would be a big effort for not much gain?

Can't. See the url above. It only can URL + `convert=%s`. One currency only unless we want to do multiple currencies + multiple calls. This currently works with `usd` + `any_other_currency`.

>It currently fails silently for ${price_usd} £{price_gbp} but would work if they are reversed.

The way it works right now, the `post_config_hook` part checks for the first 3 known possible `_currency` variables, then grab the first available one in the `format_coin`, break because of the answer above (We convert only 1).

>yeah this is hard €10 or 10€ can both be correct depending where you are.

I had currency map, then I thought, maybe it's better if we just leave them to the users instead... and to use mostly raw data (if possible). So bye, it goes.

Will add example of using gbp.